automation/jenkins_build.sh

```diff
@@ -52,9 +52,13 @@ DEVICE_TYPE_JSON=$WORKSPACE/$MACHINE.json
 VERSION_HOSTOS=$(cat $WORKSPACE/build/tmp/deploy/images/$MACHINE/VERSION_HOSTOS)
 
 DEPLOY_ARTIFACT=$(jq --raw-output '.yocto.deployArtifact' $DEVICE_TYPE_JSON)
+COMPRESSED=$(jq --raw-output '.yocto.compressed' $DEVICE_TYPE_JSON)
 mkdir -p $BUILD_DEPLOY_DIR
 rm -rf $BUILD_DEPLOY_DIR/* # do we have anything there?
 mv -v $(readlink --canonicalize $WORKSPACE/build/tmp/deploy/images/$MACHINE/$DEPLOY_ARTIFACT) $BUILD_DEPLOY_DIR/$DEPLOY_ARTIFACT
+if [ "${COMPRESSED}" == 'true' ]; then
+	gzip $BUILD_DEPLOY_DIR/$DEPLOY_ARTIFACT
+fi
```

Comment:
Fixed, and added an edison test PR at https://github.com/resin-os/resin-edison/pull/12